.github/workflows/cargo-audit.yaml

```diff
@@ -0,0 +1,24 @@
+name: Cargo Audit
+
+on:
+  push:
+    paths:
+      - '.github/workflows/cargo-audit.yaml'
+      - '**/Cargo.toml'
+      - '**/Cargo.lock'
+  pull_request:
+    paths:
+      - '.github/workflows/cargo-audit.yaml'
+      - '**/Cargo.toml'
+      - '**/Cargo.lock'
+  schedule:
+    - cron: '0 0 * * *'
+  workflow_dispatch:
+
+jobs:
+  audit:
+    runs-on: ubuntu-latest
```

Comment:
We run on `lab` by default, if I remember correctly? Double-check with @Frostman maybe